src/murmur/ServerUser.h

```diff
@@ -52,7 +52,7 @@ struct BandwidthRecord {
 
 struct WhisperTarget {
 	struct Channel {
-		int iId;
+		unsigned int iId;
```

Comment:
Can we do this in a separate commit?

Reply:
Sure we can, but why? To me this is also just a refactoring 👀

Reply:
It's not related to the changes mentioned in this PR though.

Reply:
It's part of the whisper cache creation refactoring :shrug:

Reply:
Given that the different signedness is not causing any bug (that I am aware of), I wouldn't call it a fix.

Anyway, I now have added a dedicated refactor commit that cleaned up the other members in the struct as well.

Reply:
Excellent.